internal/run/delete.go

```diff
@@ -0,0 +1,52 @@
+package run
+
+import (
+	"fmt"
+)
+
+// Delete is an execution step that calls `helm upgrade` when executed.
+type Delete struct {
+	Release string
+	DryRun  bool
+
```

Comment:
Minor: weird formatting here

Reply:
I've been in the habit of putting a blank line between public and private struct fields, because I thought it was a golang style convention. I tried to find a citation just now, though, and it looks like I imagined that rule.

It looks especially weird in [a struct with only two fields](https://github.com/pelotech/drone-helm3/blob/30e1e3b99f8e1e475ce40bc8707a32e7f51e62b1/internal/run/lint.go#L8-L12)! I'll change it.